readthedocs/search/tests/test_api.py

```diff
@@ -12,8 +12,8 @@
 @pytest.mark.search
 class TestDocumentSearch(object):
 
-    def __init__(self):
-        # This reverse needs to be inside the ``__init__`` method because from
+    def setUp(self):
```

Comment:
```
search/tests/test_api.py:13
  /Users/eric/projects/readthedocs.org/readthedocs/search/tests/test_api.py:13: PytestWarning: cannot collect test class 'TestDocumentSearch' because it has a __init__ constructor
```